package.json

```diff
@@ -120,5 +122,41 @@
       }
     },
     "helpClass": "./src/services/twilio-help/custom-help"
+  },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "github_action_changes",
+        "prerelease": "rc"
+      }
+    ],
```

Comment:
[Q] Couldn't get this, why is it needed?

Reply:
This is to do a `prerelease` on `github_action_changes` branch with `-rc` suffix. Once testing is done we can remove this change.